### DISCUSSION TITLE
Can't connect to my google account

### DISCUSSION THREAD

**Discussion by Djegger** [Floorp-Projects/Floorp#754] [Q&A]
Hi everyone ! 

First of all thanks for taking the time to read my post. 
I'm not sure if this is the right place to talk about this please correct me if i'm mistaken.
I've encountered an issue and can't connect to my google account at all. 
I'm using the latest version of Floorp which is 11.8.1. And my OS is Windows 11. 
This is the message i'm getting : This browser or app may not be secure. 
Try using a different browser. If you’re already using a supported browser, you can try again to sign in.

Do you guys have any idea of how to fix it ? Has anyone already encountered this ? 
I tried searching for it but couldn't find any fix.

Thanks again for reading this !

**Comment by jhub54** [comment]
Me too!!!

**Comment by creeper-0910** [comment]
Basically, the problem occurs when the UserAgent is disguised.

**Reply by Djegger** [reply]
Hi,
Sorry for the late answer could you explain me how to solve it ?

Thanks a lot.

**Reply by creeper-0910** [reply]
If you are using User.js, disable it.

Since there is a UserAgent switching item in the settings, please return it to default.
If it is still the default, change it to a different UserAgent and then change it back to the default again.

**Comment by jhub54** [comment]
How do you fix it?

**Comment by jhub54** [comment]
I gave up on Floorp. 

⁣
Thanks,
Jeff

Get BlueMail for Android ​

**Comment by Ryosuke-Asano** [comment]
![image](https://github.com/Floorp-Projects/Floorp/assets/73892113/0c268be4-3026-4934-ad12-840c609ff5c2)
